config/jobs/kubernetes/sig-arch/kubernetes-code-organization.yaml

```diff
@@ -74,7 +74,10 @@ presubmits:
           allowPrivilegeEscalation: false
         command:
         - runner.sh
-        - ./../test-infra/experiment/dependencies/update-dependencies-and-run-tests.sh --test-mode unit
+        - bash
+        - -c
+        - |
+          ./../test-infra/experiment/dependencies/update-dependencies-and-run-tests.sh --test-mode unit
```

Comment:
Instead of wrapping with another shell process, you can move --test-mode unit to be separate args

Reply:
exactly what i was trying to do and got wrong :) 

let's try again. updated like you asked @BenTheElder